### PR TITLE
Hauki 222 loading indicator

### PR DESCRIPTION
**Pull request by minevala**
- Move loading of date-periods inside opening-hour component and add loading and error state indicators/texts.

Näyttökuva 2020-11-13 kello 10 48 04

Näyttökuva 2020-11-13 kello 10 48 27

### CODE REVIEW

**Comment by VilleMiekkoja** [comment]
> * Move loading of date-periods inside opening-hour component and add loading and error state indicators/texts.
> 
> Näyttökuva 2020-11-13 kello 10 48 04
> 


Not sure if the loading indicator stands out enough for people to notice it? Though not that big of a deal. Would it stand out more if we horizontally center it?

File: src/resource/page/ResourcePage.tsx [hunk 1]
```diff
@@ -152,9 +135,11 @@ export default function ResourcePage({ id }: { id: string }): JSX.Element {
         </p>
       </ResourceDetailsSection>
       <ResourceSourceLink id="resource-source-link" resource={resource} />
-      <ResourceSection id="resource-opening-hours">
-        <ResourceOpeningHours id={id} datePeriods={datePeriods} />
-      </ResourceSection>
+      {resource && (
+        <ResourceSection id="resource-opening-hours">
```
Comment (VilleMiekkoja) [inline src/resource/page/ResourcePage.tsx:139]: Not related to this PR, but I think the ratio of boilerplate/beef in this `ResourceSection` component is quite large. It basically returns html tag section with id and classname. Though it is applied in 2 place so I guess having it as a separate component is justified 🤔

**Comment by VilleMiekkoja** [comment]
Good improvement! 💪

**Comment by minevala** [comment]
I could first merge the HDS-library update PR. The we could use the new spinner

**Pull request closed without merging**